ecommerce-product-page-main/styled.css

```diff
@@ -301,61 +265,63 @@ ul.list li a:hover{
 }
 
 
-
-
-
-
-
-
 .block2{
-
     flex: 1;
     margin-left: 60px;
-    
-    
+
 }
 
 
 .h5orange{
-
-        color: hsl(26,100%,55%);
-        width: 150px
+    color: hsl(26,100%,55%);
 }
 
 .h1font{
-
     font-family: 'Kumbh Sans', sans-serif;
     font-weight: 700;
-    width: 250px;
-    display: inline;
 }
 
-.pfont{
-    
+.description{
     font-family: 'Kumbh Sans', sans-serif;
     font-weight: 400;
     font-size: 13px;
     color: hsl(219, 9%, 45%);
-    width: 250px;
-    display: inline;
     
 }
 
-.containflex{
-
+.counterSection{
     display: flex;
-    
+    align-items: center;
+    gap: 12px;

```

Comment:
This gap is it for the content inside this div container?

Reply:
it is for spacing element inside the div. read up on gap in flex box